prow/plugins/approve/approvers/README.md

```diff
@@ -169,13 +176,15 @@ K8s-bot updates comment:
 
 The **lgtm** label is immediately added to the PR.
 
-**Step 3**: *approver2* writes `/approve`
+### Step 5:
+
+*approver2* writes `/approve`
```

Comment:
and is in the G OWNERS file.

Reply:
I gave approver2 OWNERS on a parent, to highlight that possibility.